js/ui/codingManager.js

```diff
@@ -352,15 +356,17 @@ const CodingManager = new Lang.Class({
         return null;
     },
 
+    _connectButtonPositioningSignalsToBuilder: function(session, builderWindow) {
```

Comment:
These functions could use a shorter name... how about `_connectBuilderSizeAndPosition()`? (and disconnect)